src/pages/Admin/Venue/Rooms/RoomsForm.tsx

```diff
@@ -101,7 +101,7 @@ interface RoomInnerForm {
   editingRoomIndex?: number;
 }
 
-export type FormValues = Partial<Yup.InferType<typeof validationSchema>>;
+export type FormValues = Yup.InferType<typeof validationSchema>;
```

Comment:
Why did we change the type from using `Partial` here? Is that correct? Will it break anything? Are the form values always guaranteed to be set in the `onSubmit`? Or does this just end up making our types 'lie' about the underlying data at 'compile time', which will then lead to hard to debug/identify runtime issues.

Reply:
All fields in the `RoomsForm` are required and there is no sense to make them partial.
And it fixed the type checking `onSubmit` with `RoomInput`. Guess now it's more secure

Reply:
So long as everything still works as expected if I only partially fill out the form, then sounds good.